### PR TITLE
Custom tag class

### PR DESCRIPTION
**Pull request by ranzem**
Contribution guide: https://github.com/calvinlough/ember-tag-input/blob/0ff659978c6dfac46500faae37f7511044831c8e/CONTRIBUTING.md

TL;DR: Pulled down a fork of ember-tag-input and updated https://github.com/calvinlough/ember-tag-input/pull/221 to get on top of the current master

Since there's a chance that even with the updates the PR will stay in stasis forever, and just to get more on eyes on it before I submit it to the main repo, thought I'd ask y'all to take a look.

The only actual change is that one line in tag-input, the rest is test/doc updating.

A corresponding ALI PR will come out soon (TM).

### CODE REVIEW

File: tests/dummy/app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.red {
+  background-color: rgb(161, 40, 40);
+}
+.green {
+  background-color: rgb(40, 161, 40);
+}
+.purple {
```
Comment (apellerano-pw) [inline tests/dummy/app/styles/app.css:7]: blue like

**Review by apellerano-pw — APPROVED** [review]

**Review by brigham-hausman — APPROVED** [review]

File: addon/components/tag-input.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{#each this.tags as |tag index|~}}
-  <li class="emberTagInput-tag">
+  <li class="emberTagInput-tag {{tag.modifiers}}">
```
Comment (jcalandro) [inline addon/components/tag-input.hbs:2]: I'm wondering if the term "modifiers" is confusing here when it is really meant to hold custom classnames 🤔 

With he current name, I would almost expect it to represent an array of ember-modifier methods or something.

**Review by jcalandro — APPROVED** [review]

File: addon/components/tag-input.hbs (hunk 2 above)
Comment (jcalandro) [reply addon/components/tag-input.hbs:2]: I guess the distinction is that this is a classname modifier vs. an element modifier.

Comment (ranzem) [reply addon/components/tag-input.hbs:2]: yeah, I'm not a huge fan of it myself - but with not changing that this is really just a refresh of an existing PR with none of my opinion bleeding into it.

I probably would have gone with `classes`

Comment (ranzem) [reply addon/components/tag-input.hbs:2]: I guess if the PR never gets accepted we can come back and play more freely

**Pull request merged**